code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -24,11 +24,12 @@ const CourseProgress = React.createClass({
   propTypes: {
     professionalLearningCourse: React.PropTypes.bool,
     focusAreaPositions: React.PropTypes.arrayOf(React.PropTypes.number),
-    stages: React.PropTypes.arrayOf(stageShape)
+    stages: React.PropTypes.arrayOf(stageShape),
```

Comment:
Nit: we've been avoiding trailing commas, but I'm not sure there's a good reason.  FWIW https://github.com/airbnb/javascript#commas recommends trailing commas everywhere.